install_config/persistent_storage/persistent_storage_azure.adoc

```diff
@@ -131,6 +131,16 @@ xref:../../dev_guide/persistent_volumes.adoc#dev-guide-persistent-volumes[reques
 storage using persistent volume claims], which can now use your new persistent
 volume.
 
+[IMPORTANT]
+====
+For a pod that has a mounted volume through an Azure disk PVC, scheduling the
+pod  to a new node takes few minutes. Wait for two to three minutes to complete
```

Comment:
extra space between "pod" and "to"

s/takes few minutes/takes a few minutes

extra space after "creation"